### ISSUE TITLE
vim process cpu 100%

### ISSUE THREAD

**Issue by henices** [Shougo/unite.vim#880]
* Problems summary
 Use unite in large project (Android kernel) cause cpu 100%

* Expected
 Use Unite.vim smoothly

* Reproducable ways from Vim starting

0. :Unite file_rec/async
1. enter i to insert mode
2. enter words to search
3. enter Esc to normal mode
4. use j to move down

 After step 5,  vim process cpu 100%

* Environment Information
 * OS:                Fedora
 * Vim version:  7.4.475 
 * Enabled lua interface?: 

```+acl             +farsi           +mouse_netterm   +syntax
+arabic          +file_in_path    +mouse_sgr       +tag_binary
+autocmd         +find_in_path    -mouse_sysmouse  +tag_old_static
-balloon_eval    +float           +mouse_urxvt     -tag_any_white
-browse          +folding         +mouse_xterm     -tcl
++builtin_terms  -footer          +multi_byte      +terminfo
+byte_offset     +fork()          +multi_lang      +termresponse
+cindent         +gettext         -mzscheme        +textobjects
-clientserver    -hangul_input    +netbeans_intg   +title
-clipboard       +iconv           +path_extra      -toolbar
+cmdline_compl   +insert_expand   +perl            +user_commands
+cmdline_hist    +jumplist        +persistent_undo +vertsplit
+cmdline_info    +keymap          +postscript      +virtualedit
+comments        +langmap         +printer         +visual
+conceal         +libcall         +profile         +visualextra
+cryptv          +linebreak       +python/dyn      +viminfo
+cscope          +lispindent      -python3         +vreplace
+cursorbind      +listcmds        +quickfix        +wildignore
+cursorshape     +localmap        +reltime         +wildmenu
+dialog_con      +lua/dyn         +rightleft       +windows
+diff            +menu            +ruby/dyn        +writebackup
+digraphs        +mksession       +scrollbind      -X11
-dnd             +modify_fname    +signs           -xfontset
-ebcdic          +mouse           +smartindent     -xim
+emacs_tags      -mouseshape      -sniff           -xsmp
+eval            +mouse_dec       +startuptime     -xterm_clipboard
+ex_extra        +mouse_gpm       +statusline      -xterm_save
+extra_search    -mouse_jsbterm   -sun_workshop    -xpm```

**Comment by Shougo** [comment]
#723 

I think the problem is fixed in Vim 7.4.615.
Please update your Vim.

**Comment by henices** [comment]
Update to Vim 7.4.615,  unite.vim works well. Thank you for your reply.
